url.bs

```diff
@@ -787,9 +788,15 @@ then runs these steps:
 <var>input</var>, and then runs these steps:
 
 <ol>
- <li><p>If <var>input</var> contains a <a>forbidden host code point</a> excluding U+0025 (%),
+ <li><p>If <var>input</var> contains a <a>forbidden host code point</a>, excluding U+0025 (%),
```

Comment:
IMO this comma addition slightly decreases readability.

Reply:
So one usage of excluding could also omit it, but another is after inclusive and therefore cannot omit it. I guess I'm okay with omitting when possible.